### ISSUE TITLE
SRA>System settings>Organizations: Editing an organization information create a new record in the table 

### ISSUE THREAD

**Issue by SteeveEbener** [terraframe/geoprism-registry#200]
- CGR version: 0.7.0

**Describe the bug (clear and concise)**
Editing an organization information actually make a new record appear in the window:
![Org_new_rec](https://user-images.githubusercontent.com/12602163/80768578-5dda9380-8b7d-11ea-9db1-35c58537e4b9.jpg)

The old record disappear if you refresh the browser. 

If you don't refresh and you try to delete the "old" record  you actually delete the organization in question

**To Reproduce**
As a SRA
1. Go to System settings
2. Click on the editing pen for an existing organization
3. Change one of the information (Name, contact)
4. Save the change
3. See error: new record appearing in the table 

**Expected behavior (clear and concise)**
Editing the information attached to an organization does not create a new record in the table

**Screenshots**
See above

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Google Chrome
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Additional context (if any)**

**Comment by CarlyNel** [comment]
Hi @SteeveEbener, I've just tested this and it's working as expected on my side. I'm logged in as an SRA, followed your above instructions, made edits, saved my changes and everything is reflecting correctly without me having to refresh my browser. Screenshot below. I'm on windows 10, using Firefox 77.0.1 (64-bit) (which I just updated).

![image.png](https://images.zenhubusercontent.com/5eda94a201596292f642f925/0f08d4cd-f28e-4c41-9be2-39b3967328ec)
